public/js/admin.js

```diff
@@ -265,32 +279,73 @@ $('#option-settings').click(function(evt) {
 });
 
 /*
- * Process user deletion request.
- * First, display a confirmation message and then request the user's removal.
+ * Process user deactiavtion request.
+ * First, display a confirmation message and then request the user's deactivation.
  */
-var deleteUser = function(id) {
+var deactivateUser = function(id) {
     swal({
-        title: 'Confirm deletion',
-        text: 'User ' + id + ' will be removed from the database.',
+        title: 'Confirm Deactivation',
+        text: id + '\'s  account will be deactivated.',
         type: 'warning',
         showCancelButton: true,
         confirmButtonColor: '#DD6B55',
-        confirmButtonText: 'Delete',
+        confirmButtonText: 'Deactivate',
         closeOnConfirm: true
     }, function() {
         $.ajax({
             type: 'POST',
-            url: '/userdel',
-            data: { userid: id },
+            url: '/setUserStatus',
+            data: { userid: id, active: false },
             success: function(data) {
                 displayAccountsTable();
-                dropSnack(colours.SUCCESS, 'User ' + id + ' was removed from the database');
+                const msg = ' has been&nbsp;<u><b>deactivated</b></u>&nbsp;';
+                const colour = colours.WARNING;
+                const icon = '<i class="material-icons">warning</i>&nbsp&nbsp&nbsp';
```

Comment:
I see the use of &nbsp&nbsp&nbsp in a lot of places. Is there a better way of doing this?

Reply:
we can make a function to generate this

Reply:
thats for Mo, not related to this PR